dashboard/app/models/script.rb

```diff
@@ -1626,4 +1626,8 @@ def self.has_any_pilot_access?(user = nil)
     return true if user.permission?(UserPermission::LEVELBUILDER)
     all_scripts.any? {|script| script.has_pilot_experiment?(user)}
   end
+
+  def self.script_names_by_curriculum_umbrella(curriculum_umbrella)
+    Script.all.select {|script| script.curriculum_umbrella == curriculum_umbrella}.pluck(:name)
```

Comment:
rather than having to grab every script and filter from there, there is some funky ActiveRecord syntax we can use to grab only scripts that have the matching curriculum_umbrella in their properties:
```
Script.where("properties -> '$.curriculum_umbrella' = ?", curriculum_umbrella).pluck(:name)
```

Reply:
TIL! Thanks, Maddie!